src/main/java/nl/tudelft/lifetiles/traverser/models/UnifiedPositionTraverser.java

```diff
@@ -0,0 +1,65 @@
+package nl.tudelft.lifetiles.traverser.models;
+
+import nl.tudelft.lifetiles.graph.models.Edge;
+import nl.tudelft.lifetiles.graph.models.Graph;
+import nl.tudelft.lifetiles.graph.models.sequence.SequenceSegment;
+
+/**
+ * UnifiedPositionTraverser which generates unified positions for each graph so
+ * all vertices independent of it's reference in the original are unified within
+ * the same coordinate system.
+ *
+ * @author Jos
+ *
+ */
+public class UnifiedPositionTraverser {
+    /**
+     * Graph which is being traversed.
+     */
+    private Graph<SequenceSegment> graphVar;
+
+    /**
+     * Traverses and copies a graph.
+     *
+     * @param graph
+     *            Graph to be traversed, calculate unified positions.
+     * @return traversed copy of the graph.
+     */
+    public final Graph<SequenceSegment> traverseGraph(
```

Comment:
Should rename this to be more descriptive. Basically, you removed the graphtraverser Interface, but you didn't actually change the implementations. They're still implicitly following the same interface.